front/app/components/admin/UserFieldsInSurveyToggle/UserFieldsInSurveyToggle.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import { Toggle, Text, Box } from '@citizenlab/cl2-component-library';
+
+import useFeatureFlag from 'hooks/useFeatureFlag';
+
+import { SectionField, SubSectionTitle } from 'components/admin/Section';
+
+import { FormattedMessage } from 'utils/cl-intl';
+
+import messages from './messages';
+
+interface UserFieldsInSurveyToggleProps {
+  user_fields_in_form: boolean | null | undefined;
```

Comment:
Could we call this prop `userFieldsInForm` instead? Just matches our conventions a bit better :)

Reply:
Then you can use:

`userFieldsInForm?: boolean | null;`